src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -794,10 +799,24 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                                               uPerpTime)
                      call mpas_pool_get_array(statePool, 'normalBarotropicVelocitySubcycle', normalBarotropicVelocitySubcycleNew, &
                                               newBtrSubcycleTime)
-                     call mpas_pool_get_array(statePool, 'sshSubcycle', sshSubcycleCur, oldBtrSubcycleTime)
 
                      call mpas_pool_get_array(diagnosticsPool, 'barotropicForcing', barotropicForcing)
 
+                     ! Subtract tidal potential from ssh, if needed
+                     if (config_use_tidal_potential_forcing) then
+                       call mpas_pool_get_array(statePool, 'sshSubcycle', sshSubcycleCurNoTides, oldBtrSubcycleTime)
+                       call mpas_pool_get_array(forcingPool, 'sshSubcycleCurWithTides', sshSubcycleCur)
```

Comment:
I find the discrepancy between the names of the local pointers and the variables in the pool pretty confusing.  I grepped the code for sshSubcycleNoTides trying to see where that got updated only to realize that's not the name of it elsewhere.  In general, it helps to keep the names of local pointers as consistent as possible with the names in pools.

Reply:
I took care of this the best I could in my latest commits.